docs/authentication/session-management.mdx

```diff
@@ -0,0 +1,896 @@
+---
+title: Session Management
+id: session-management
+description: ''
+slug: /session-management
+keywords:
+  - api token
+  - access token
+  - session management
+pagination_next: null
+pagination_prev: null
+last_update:
+  date: 08/08/2023
+  author: William May
+doc_type: overview
+displayed_sidebar: mainSidebar
+---
+
+import MultiLanguageCodeBlock from '@site/src/components/CodeBlocks/MultiLanguageCodeBlock';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## What is Session Management
+
+Session management is a crucial aspect of web application development that involves securely managing user sessions. An authentication session is a period of user interaction with the application, typically starting with the user providing credentials and ending with logout or inactivity timeout. Effective session management ensures user data privacy, maintains session state, and provides a seamless user experience.
+
+The Beyond Identity session consists of something we call a `token`, specifically access tokens. An access token, also called an API token, is an object or file that enables access to an application or resource. The token is digitally signed and contains information about what resources can be accessed and for how long.
```

Comment:
does the session consist also of [optionally] refresh tokens?

Reply:
I believe so, since they can be used to extend a session. Nikhil and Tomas are out this week, but I added/copied a short blurb about refresh tokens.